vignettes/cookbook_advanced_use_cases.Rmd

```diff
@@ -14,10 +14,520 @@ knitr::opts_chunk$set(
+
+The types of use cases we explore in this section all build on the standard PACTA for Banks outputs – that is they use results based on the market share approach or the sectoral decarbonization approach – either at loan book level or at company level. They vary by the intended target audience – generally either an internal audience (another department in the FI or management) or an external audience (often either a client or a regulator/supervisor, sometimes the broader public). Additionally, the use cases vary by type, where some are focused more on communicating/reporting and others are more focused on implementing change or informing internal decisions.
+
+We will explain in some detail some of the most established use cases in this cookbook, but we will mention others only briefly, especially ones that have not been explored in quite as much detail yet and that would therefore require additional research before describing potential ways to implement them.
+
+## Use Case 1: Regulatory compliance
+
+### Description
+
+Report or disclose loan book alignment or targets to a financial supervisor or regulator
+
+### Level of analysis
+
+Regulators will normally not mandate reporting on individual clients and normally leave the choice of clients up to the financial institution. Instead, rules and regulations apply to the structure of the overall loan book. Hence, in most cases, calculating PACTA for Banks results at the loan book level will be the appropriate depth of analysis for this use case.
+
+### Potential application
+
+**Overview**
+
+Financial regulators across a variety of jurisdictions in the world have introduced regulations to account for climate-related financial risks in recent years. The exact specifications of regulatory requirements differ on a case-by-case basis. But in many cases, the regulations involve reporting on the materiality of climate-relevant exposures and in some cases they include reporting on forward-looking metrics, such as alignment of emissions or production to an external benchmark or emissions/production targets of the financial institution to ensure climate transition trends are accounted for by financial institutions. In all cases, requirements can vary by type of climate metric to report on (e.g. absolute emissions, financed emissions, production-based emission intensity, production volumes), sectoral scope and granularity (financial scope and granularity: all exposures combined at the loan book level, aggregate exposures to clients in climate-relevant sectors only, sector-level exposures to climate relevant sectors; sectoral scope: reporting on all parts of the value chain or on the ones most relevant for emissions).
```

Comment:
> Financial regulators across a variety of jurisdictions in the world have introduced regulations to account for climate-related financial risks in recent years

Would be good to give at least once example here, if we have one? I wonder if referencing the EBA makes sense here?

Reply:
Valid point, I do think it needs careful wording though, as I think we don't want to say that we are sure that PACTA will be sufficient when reporting to the EBA. Maybe good to sync with George on this topic

Reply:
Hmmm ok. Let's not let that block this PR then, and we can consider that a potential addition if we are confident we can do so 😊